ios/BT/API/Model/ExposureConfiguration.swift

```diff
@@ -16,7 +16,7 @@ extension ExposureConfiguration {
 
   static var placeholder: ExposureConfiguration = {
     ExposureConfiguration(minimumRiskScore: 0,
-                          attenuationDurationThresholds: [50, 70],
+                          attenuationDurationThresholds: [53, 60],
```

Comment:
these are very specific 😅  could we put them in named constants for "documentation" purposes?